DIGNDB.App.SmitteStop.API/Services/RollingKeyPairGeneratorWithLocalCertificate.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DIGNDB.App.SmitteStop.API.Attributes;
+using DIGNDB.App.SmitteStop.API.Services;
+using Org.BouncyCastle.Asn1.X9;
+using Org.BouncyCastle.Crypto.EC;
+using Org.BouncyCastle.Math;
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace DIGNDB.App.SmitteStop.API
```

Comment:
Namespaces should match directory structure.
In general it should be `DIGNDB.App.SmitteStop.API/Services`

Reply:
I fixed this now. Feeling a bit ashamed here. (In my defence, I'm used to Java where the tools fix this automatically and if you somehow make a mistake, the compiler stops you)